### ISSUE TITLE
Execute tests in parallel

### ISSUE THREAD

**Issue by tkroman** [scalacenter/bloop#903]
Nice to have: executing tests for separate projects in parallel. Should be done optionally at user's discretion (e.g. some integration tests probably can't even be run in parallel, but most unit tests can be).
As fas as I can tell, compilation phase handling is already delegated to compilation task which was solved by #11, so we should do something similar for tests.

cc @jvican

**Comment by jvican** [comment]
Fixed by #935, thank you @tkroman :pray:
